bitrise.yml

```diff
@@ -1470,7 +1471,7 @@ workflows:
                 exit 1
             fi
 
-            if [[ ! -d "${PRODUCTS_DIR}/${SCHEME}-${SDK}" ]]; then
+            if [[ ! -d "${PRODUCTS_DIR}/${SCHEME}-iphoneos" ]]; then
```

Comment:
Same here, see comment above, so this wouldn't need to be hardcoded.

Reply:
Agree. The file/directory ends with "iphoneos" and not "iphoneos18.2", so I can't just use the `$SDK` variable. 😞

Reply:
I could use another variable to denote the xcode version. 🤔